features/step_definitions/record_of_guidance_contents_steps.rb

```diff
@@ -97,35 +106,45 @@
   end
 end
 
-# Then(/^the "pension pot" section should be the "(.*?)" version$/) do |version|
-#   version = case version
-#             when 'only their DC pot and state pension' then 'pension'
-#             when 'multiple sources'                    then 'other'
-#             end
+Then(/^the "pension pot" section should be the "(.*?)" version$/) do |version|
+  version = case version
+            when 'only their DC pot and state pension' then 'pension'
+            when 'multiple sources'                    then 'other'
+            end
 
-#   expect(page).to include_output_document_section('pension pot').at_version(version)
-# end
+  expect_uploaded_csv_to_include(income_in_retirement: version)
+end
 
 Given(/^(?:I|we) have captured the customer's details in an appointment summary$/) do
   @appointment_summary = fixture(:populated_appointment_summary)
 end
 
-# Then(/^the record of guidance should include their details$/) do
-#   output_document = OutputDocument.new(@appointment_summary)
+Then(/^the record of guidance should include their details$/) do
+  output_document = OutputDocument.new(@appointment_summary)
+
+  expected = {
+    attendee_name: output_document.attendee_name,
+    value_of_pension_pots: output_document.value_of_pension_pots
+  }
 
-#   expect(page).to have_content(output_document.attendee_name)
-#   expect(page).to have_content(output_document.value_of_pension_pots)
-# end
+  expect_uploaded_csv_to_include(expected)
+end
 
 Given(/^(?:I|we) have captured appointment details in an appointment summary$/) do
   @appointment_summary = fixture(:populated_appointment_summary)
 end
 
-# Then(/^the record of guidance should include the details of the appointment$/) do
-#   output_document = OutputDocument.new(@appointment_summary)
+Then(/^the record of guidance should include the details of the appointment$/) do
+  output_document = OutputDocument.new(@appointment_summary)
 
-#   expect(page).to have_content(output_document.appointment_date)
-#   expect(page).to have_content(output_document.appointment_reference)
-#   expect(page).to have_content(output_document.guider_first_name)
-#   expect(page).to have_content(output_document.guider_organisation)
-# end
+  expected = {
```

Comment:
The lead isn't actually in the record of guidance, only the covering letter, so the assertion is part of that feature.

Somewhat related: I've raised an issue (#108), for us to review the naming of the various documents to make sure its consistent and clear.